### PR TITLE
[WIP] updated changelog for 2.4.0

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
-    - server-room,
-    - rack,
-    - position,
-    - orientation
-    - slot-number,
-
+New features
+************
+* Added Region feature to assets, liceces and supports.
+* Added location fields to device_info model: data-center, server-room, rack, position, slot, orientation, slot-number.
```
Comment (xliiv) [inline CHANGES.rst]: "Added location fields to data center asset: data-center, server-room, rack, position, slot, orientation, slot-number."?

File: CHANGES.rst [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
-* Added feature Region to assets, liceces and supports,
-* Added new field *segment* to back office assets and user-details,
-* Created new widget dedicated to many-to-many relations,
-* Reduced SQL queries about 50% in ``Licence`` and ``Hardware`` list view,
-* Moved licence module to new app structure,
-* Cleaned up urls.py.
+Fix
+***
+* Fixed soft-deleting feature on Licenses and Supports.
+* Sync parent also for blade servers - use slote_no instead position.
```
Comment (xliiv) [inline CHANGES.rst]: add new item, please:
"Fixed counting in 'status - model' report"

**Comment by vi4m** [comment]
@mkurek ping

**Comment by mkurek** [comment]
@vi4m done

**Comment by vi4m** [comment]
LGTM

**Comment by vi4m** [comment]
LGTM

**Pull request merged**